### PR TITLE
Remove BBB code and imports for Zope < 2.13.

### PR DESCRIPTION
**Pull request by thet**
/cc @do3cc

### CODE REVIEW

**Comment by thet** [comment]
The plone.testing tests pass. The Jenkins tests fail, of course, because of a lot of test isolation problems.

@do3cc can you have a look if this PR goes OK for you? We don't need special handling of Zope < 2.13 for plone.testing anymore.

**Comment by do3cc** [comment]
Ah, took me a while to understand why the test isolation fails break through :-)

File: src/plone/testing/z2.py [hunk 1]
```diff
@@ -554,10 +527,8 @@ def setUpDatabase(self):
         that the database that is opened by Zope 2 is in fact the top of
         the resource stack.
         """
-
-        if HAS_ZOPETESTCASE:
-            from Testing.ZopeTestCase.ZopeLite import _patched as ZOPETESTCASEALERT
-        if HAS_ZOPETESTCASE and ZOPETESTCASEALERT:
+        from Testing.ZopeTestCase.ZopeLite import _patched as ZOPETESTCASEALERT
```
Comment (do3cc) [inline src/plone/testing/z2.py]: This import is not conditional any more and should be moved to the top

File: src/plone/testing/zca.py [hunk 2]
```diff
@@ -31,11 +31,11 @@ def _hookRegistry(reg):
     globalregistry.base = reg
     globalregistry.globalSiteManager = reg
 
-    # Set the default global site manager for new threads when zope.site's
+    # Set the default global site manager for new threads when zope.component
     # hooks are in place
 
     try:
-        from zope.site.hooks import SiteInfo
+        from zope.component.hooks import SiteInfo
```
Comment (do3cc) [inline src/plone/testing/zca.py]: Can you check,. if this can ever fail?
Same for the next try except catcher

**Comment by do3cc** [comment]
Apart from the minor nitpicks, it looks good

File: src/plone/testing/zca.py (hunk 2 above)
Comment (thet) [inline src/plone/testing/zca.py]: zope.component cannot fail. But the next one, five.localsitemanager, might simply not be available...

**Comment by thet** [comment]
Tnx for review! I'm going to merge...

**Pull request merged**